setup.py

```diff
@@ -50,7 +50,10 @@ def find_packages(where='.', lib_prefix='', exclude=()):
     url='https://github.com/mozilla/jx-sqlite',
     license='MPL 2.0',
     packages=find_packages('.', lib_prefix=''),
-    install_requires=["future", "mo-collections>=1.2.17235", "mo-dots>=1.5.17188", "mo-files>=1.2", "mo-json-config", "mo-json>=1.0.17168", "mo-kwargs", "mo-logs", "mo-math", "mo-testing>=1.0.17168", "mo-threads", "mo-times"],
+    install_requires=["future", "mo-collections>=1.2.17235", "mo-dots>=1.5.17188",
+                      "mo-files>=1.2", "mo-json-config", "mo-json>=1.0.17168",
+                      "mo-kwargs", "mo-logs", "mo-math", "mo-testing>=1.0.17168",
+                      "mo-threads", "mo-times"],
```

Comment:
could you make these one requirement per line:
install-requires=["future",
                            "mo-collections>=1.2.17235",
                           ...],

Reply:
Will do it.